deployment-template/eks/eks-ci-daemonset.yaml

```diff
@@ -0,0 +1,252 @@
+# create namespace
```

Comment:
Lets align the CI deployment template files location with PR https://github.com/aws-observability/aws-otel-collector/pull/453/files